orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/Unique.feature

```diff
@@ -61,12 +61,12 @@ Feature: As a user
 
   Scenario: The one where combination strategy is exhaustive
     Given the combination strategy is exhaustive
-    And there is a field foo
+    And there is a non nullable field foo
```

Comment:
would it be better to just have this be 
'there is a field'
rather than 
'there is a non nullable field'

as it's non nullable by default

Reply:
I did think about this but I thought it was better to be explicit in the tests, especially since then the test should highlight anywhere the behaviour is wrong (haven't spotted any yet)